src/main/java/hudson/plugins/sloccount/SloccountBuildAction.java

```diff
@@ -37,6 +45,12 @@ public String getDisplayName() {
     public String getUrlName() {
         return URL_NAME;
     }
+    
+    @Override  
+    public Collection<? extends Action> getProjectActions() {  
+        return this.projectActions;  
```

Comment:
From what I see you could just generate this list on the flight. No need to cache it